tests/lib/cli.js

```diff
@@ -1649,6 +1649,105 @@ describe("cli", () => {
             });
         });
 
+        describe("flat Only", () => {
```

Comment:
This was probably accidentally inserted in "eslintrc Only".

```
  cli
    eslintrc Only
      flat Only
        `--plugin` option
          √ should load a plugin from a CommonJS package (41ms)
          √ should load a plugin from an ESM package
          √ should load multiple plugins
          √ should resolve plugins specified with 'eslint-plugin-'
          √ should resolve plugins in the parent directory's node_module subdirectory
          √ should fail if a plugin is not found
          √ should fail if a plugin throws an error while loading
          √ should fail to load a plugin from a package without a default export
```

Reply:
Missed that, thanks! It's fixed in 34e02cb09ca0bb23d584f3ba2487715f653dd78f.